src/java.management.rmi/share/classes/javax/management/remote/rmi/RMIConnectorServer.java

```diff
@@ -132,7 +132,7 @@
      * the serial form of any deserialized object.
      * The pattern must be in same format as used in
      * {@link java.io.ObjectInputFilter.Config#createFilter}.
-     * It may define a white list of permitted classes, a black list of
+     * It may define an accept-list of permitted classes, a reject-list of
```

Comment:
should accept-list be allow-list to be consistent with the other two RMI classes and ObjectInputFilter.Status#ALLOWED?